README.md

```diff
@@ -27,7 +27,7 @@ Read a summary of what happened this past week at [*This Week in Amethyst*][tw].
 
 ## Vision
 
-Amethyst aims to be a fast, [data-oriented](https://www.amethyst.rs/book/master/html/glossary.html#data-driven-design), and data-driven game engine suitable
+Amethyst is a fast, [data-oriented](https://www.amethyst.rs/book/master/html/glossary.html#data-driven-design), and data-driven game engine suitable
```

Comment:
Do we have benchmarks already?

Reply:
I suppose we don't, but then again neither does any other game engine I've seen.

Reply:
But still, how can you be sure it actually is already?